backend/danswer/direct_qa/qa_prompts.py

```diff
@@ -24,20 +31,107 @@
 }
 
 
-def json_processor(question: str, documents: list[str]) -> str:
+def add_metadata_section(
+    prompt_current: str,
+    chunk: InferenceChunk,
+    prepend_tab: bool = False,
+    include_sep: bool = False,
+) -> str:
```

Comment:
nit: add comment about the assumptions this function is making about `prompt_current`